packages/build-utils/src/workspaces/workspace-managers.ts

```diff
@@ -0,0 +1,65 @@
+import type { Framework } from '@vercel/frameworks';
+
+/**
+ * The supported list of workspace managers.
+ *
+ * This list is designed to work with the @see {@link detectFramework} function.
+ *
+ * @example
+ *   import { workspaceManagers as frameworkList } from '@vercel/build-utils/workspaces'
+ *   import { detectFramework } from '@vercel/build-utils'
+ *
+ *   const fs = new GitDetectorFilesystem(...)
+ *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
+ *
+ * @todo Will be used by the detect-eligible-projects API endpoint for a given git url.
+ */
+export const workspaceManagers = [
```

Comment:
```suggestion
export const workspaceManagers: Array<Framework> = [
```

Reply:
Have to cast to Framework[] (or Array<Framework>) using `as` otherwise there's a bunch of type errors because of missing props.  The missing props are not needed for our use case.

This is also the same convention I found in `frameworks.ts` as well so was trying to be consistent 😅

Reply:
Hmm I'd rather see the frameworks type be updated to mark those properties as optional, or a new type be created using Omit utility to restrict the type of this export. 

As it currently stands we are asking for a future issues related to mismatching types